drake/systems/plants/RigidBodyFrame.h

```diff
@@ -3,9 +3,18 @@
 
 class DLLEXPORT_RBM RigidBodyFrame {
 public:
+  RigidBodyFrame(const std::string& _name, const std::shared_ptr<RigidBody>& _body, const Eigen::Matrix4d& _Ttree)
```

Comment:
I think that all of these need to be cleaned up in our real scrubbing pass.  The RBM code right now is so gross (ineffected with indexes instead of references, structure copies, etc) that I wanted to cry.  As long as you think this is sufficiently correct, I’d like to push and then scrub it together.

> On Aug 30, 2015, at 5:51 PM, Twan Koolen <notifications@github.com> wrote:
> 
> In drake/systems/plants/RigidBodyFrame.h <https://github.com/RobotLocomotion/drake/pull/1303#discussion_r38276408>:
> 
> > @@ -3,9 +3,18 @@
> >  
> >  class DLLEXPORT_RBM RigidBodyFrame {
> >  public:
> > +  RigidBodyFrame(const std::string& _name, const std::shared_ptr<RigidBody>& _body, const Eigen::Matrix4d& _Ttree)
> What is a Ttree? I know, but it's not going to be clear for anybody else. Use transform_to_body instead?
> why a shared_ptr for body?
> —
> Reply to this email directly or view it on GitHub <https://github.com/RobotLocomotion/drake/pull/1303/files#r38276408>.
>